EIPS/eip-1716.md

```diff
@@ -0,0 +1,38 @@
+---
+eip: 1716
+title: "Hardfork Meta: Petersburg"
+author: Marius van der Wijden (@MariusVanDerWijden)
```

Comment:
Why have you marked yourself as the author of this?

Reply:
Because pawel said so in his review. 
I do not want to step on anyone's feet. 
I just think this pr should be merged, I can change it back if you want to